lib/plenario_web/router.ex

```diff
@@ -98,6 +98,8 @@ defmodule PlenarioWeb.Router do
     resources "/etl-jobs", EtlJobController
 
     resources "/export-jobs", ExportJobController
+
+    resources "/aot", AotController
```

Comment:
So this is something we need to address with the next PR, but now we have three "special" AOT routes

- `/aot`
- `/api/v2/aot`
- /explore/array-of-things`

And I'm somewhat losing sight of the significance of them. API aot is straight forward, but are we going to have a new, separate AOT explorer that is somehow more specific than the one at `/explore/array-of-things`

Reply:
If the `/expore/array-of-things` isn't significant any more, let's just delete that route and it's associated controller actions.